examples/simple_examples/async_wss_web_server.py

```diff
@@ -0,0 +1,59 @@
+import os
+import uuid
+import random
+import json
+from aiohttp import web
+from panini import app as panini_app
+from examples.simple_examples._wss_manager import WSSManager, html
+
+
+app = panini_app.App(
+    service_name='async_NATS_WSS_bridge',
+    # host='nats-server' if 'HOSTNAME' in os.environ else '127.0.0.1',
+    host='54.36.108.188',
+    port=4222,
+    app_strategy='asyncio',
+    web_server=True,
+    web_port=1111
+)
+log = app.logger
+test_msg = {'key1': 'value1', 'key2': 2, 'key3': 3.0, 'key4': [1, 2, 3, 4], 'key5': {'1': 1, '2': 2, '3': 3, '4': 4, '5': 5},
+       'key6': {'subkey1': '1', 'subkey2': 2, '3': 3, '4': 4, '5': 5}, 'key7': None}
+
+manager = WSSManager(app)
+
+# @app.timer_task(interval=1)
+# async def publish_pereodically_for_test():
+#     test_msg['key3'] = random.random()
+#     await app.aio_publish(test_msg, subject='test.subject')
+
+
+@app.http.get('/')
+async def web_endpoint_listener(request):
+    return web.Response(text=html, content_type='text/html')
```

Comment:
It will be great, to have a comment with example of subscription
# {"subjects": ["*.>"], "action": "subscribe"} - to subscribe to all subjects

Reply:
indeed